Github actions are failing

- `tsc` action is failing with the following error `Cannot find module '@blockstack/clarity' or its corresponding type declarations.`
- `prettier` action is failing because 3 files aren't linted.

**To Reproduce**
1. `lerna clean`
2. `npm i` from root
3. `npx tsc`
4. See `tsc` error as in above description.
5. `npx prettier . -c` from root.
6. See 3 files that aren't linted (two are `.github/` files, the other is our `plugin-trait.ts` test.

**Expected behavior**
`tsc` and `prettier` actions should pass.

--- a/.prettierignore
+++ b/.prettierignore
@@ -8,3 +8,4 @@ packages/contracts/tsconfig.json
 packages/ui/.next/
 README.md
 tsconfig.json
+.github/

--- a/package.json
+++ b/package.json
@@ -8,5 +8,8 @@
     "lerna": "^3.22.1",
     "prettier": "2.3.1",
     "typescript": "^4.3.2"
+  },
+  "dependencies": {
+    "@blockstack/clarity": "^0.3.15"
   }
 }

--- a/packages/contracts/test/plugin-trait.ts
+++ b/packages/contracts/test/plugin-trait.ts
@@ -3,11 +3,15 @@ import { Client, Provider, ProviderRegistry } from "@blockstack/clarity";
 describe("plugin-trait contract test suite", () => {
   let provider: Provider;
   let client: Client;
-  
+
   describe("Contract syntax", async () => {
     it("should be valid", async () => {
       provider = await ProviderRegistry.createProvider();
-      client = new Client("SP3GWX3NE58KXHESRYE4DYQ1S31PQJTCRXB3PE9SB.plugin-trait", "plugin-trait", provider);
+      client = new Client(
+        "SP3GWX3NE58KXHESRYE4DYQ1S31PQJTCRXB3PE9SB.plugin-trait",
+        "plugin-trait",
+        provider
+      );
 
       await client.checkContract();
       await client.deployContract();

